include/cooperative_perception/detected_object.hpp

```diff
@@ -16,7 +33,9 @@ struct DetectedObject
   CovarianceType covariance;
 };
 
-using DetectedObjectType = std::variant<>;
+using VehicleObject = DetectedObject<CtrvState, CtrvStateCovariance>;
+
+using DetectedObjectType = std::variant<VehicleObject>;
 
 using DetectedObjectList = boost::container::static_vector<DetectedObjectType, 200>;
```

Comment:
The `200` is a magic number and should be moved to a compile time constant.

Reply:
Maybe `static constexpr auto kMaxDetectedObjects{ 200 };`

Reply:
Yes, so the new code would look like

```cpp
namespace cooperative_perception {
// ...
inline constexpr auto kMaxDetectedObjects{ 200U };
using DetectedObjectList = boost::container::static_vector<DetectedObjectType, kMaxDetectedObjects>;
// ...
}
```

Reply:
Cool - I updated on my local copy before I saw this to use:
`static constexpr auto kMaxDetectedObjects {200};`
I was just copying the way I saw you setup the constant for `CtrvModel` in its header. Why would we use `inline` vs `static`?